public/levels/debug-pipe.json

```diff
@@ -41,7 +41,7 @@
     "entities": [
         {
             "name": "pipe-portal",
-            "pos": [20, 174],
+            "pos": [20, 64],
```

Comment:
Whats this for?

Reply:
Hmm. Not a change I did. Must have made it into my branch from a more recent master than I did my branch off. Should not have been part of commit.